### PR TITLE
hunspellDicts.pt-pt + hunspellDicts.pt-br: init at 6.3.0.4

### PR DESCRIPTION
**Pull request by baduhai**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/hunspell/dictionaries.nix [hunk 1]
```diff
@@ -878,4 +878,26 @@ rec {
...
+  };
+
+  pt_BR = pt-br;
+  pt-br = mkDictFromLibreOffice {
+    shortName = "pt-br";
+    dictFileName = "pt_BR";
+    readmeFile = "README_pt_BR.txt";
+    shortDescription = "Brazilian Portuguese (Brazil)";
+    license = with lib.licenses; [ lgpl3 ];
+    maintainers = with maintainers; [ baduhai ];
```
[the remark this answers is not in the document]
Comment (baduhai) [reply pkgs/development/libraries/hunspell/dictionaries.nix]: I had changed this in my local brach, but never committed.

**Comment by wegank** [comment]
Under this:

https://github.com/NixOS/nixpkgs/blob/d4ec6e6fe530445d4877f997368588966eb5e8c2/pkgs/development/libraries/hunspell/dictionaries.nix#L268

add

```
 , maintainers ? with maintainers; [ vlaci ]
```

Then modify

https://github.com/NixOS/nixpkgs/blob/d4ec6e6fe530445d4877f997368588966eb5e8c2/pkgs/development/libraries/hunspell/dictionaries.nix#L287

to

```
maintainers = maintainers;
```

File: pkgs/development/libraries/hunspell/dictionaries.nix [hunk 2]
```diff
@@ -266,6 +266,7 @@ let
     , license
     , readmeFile ? "README_${dictFileName}.txt"
     , sourceRoot ? dictFileName
+    , maintainers ? with maintainers; [ vlaci ]
```
Comment (wegank) [inline pkgs/development/libraries/hunspell/dictionaries.nix:269]: Sorry again...

```suggestion
    , maintainers ? with lib.maintainers; [ vlaci ]
```

File: pkgs/development/libraries/hunspell/dictionaries.nix [hunk 3]
```diff
@@ -284,7 +285,7 @@ let
         homepage = "https://wiki.documentfoundation.org/Development/Dictionaries";
         description = "Hunspell dictionary for ${shortDescription} from LibreOffice";
         license = license;
-        maintainers = with maintainers; [ vlaci ];
+        maintainers = maintainers;
```
Comment (wegank) [inline pkgs/development/libraries/hunspell/dictionaries.nix:288]: Maybe it's better to change this to

```
inherit license maintainers;
```

File: pkgs/development/libraries/hunspell/dictionaries.nix (hunk 2 above)
Comment (baduhai) [reply pkgs/development/libraries/hunspell/dictionaries.nix]: No need to be sorry, let's get this fixed!

**Comment by wegank** [comment]
@ofborg build hunspellDicts.pt-pt hunspellDicts.pt-br

**Comment by baduhai** [comment]
@wegank can this be merged yet?

**Comment by wegank** [comment]
> @wegank can this be merged yet?

Needs to be rebased at least...

**Pull request closed without merging**